NAMESPACE

```diff
@@ -11,5 +11,6 @@ export(credentials_gce)
 export(credentials_service_account)
 export(credentials_travis)
 export(credentials_user_oauth2)
+export(tidyverse_app)
```

Comment:
Sadly, the other tidyverse packages won't be able to pass `R CMD check` and get on CRAN if we do this. It would be nice, though.